script.js

```diff
@@ -0,0 +1,13 @@
+let result;
+let number1 = prompt("What is your first number?");
+let number2 = prompt("What is your second number?");
+
+console.log(number1);
+console.log(number2);
+
+function square(number1, number2) {
+  result = number1 * number2;
```

Comment:
it would be wise to use `Number()` on these to make sure you are doing math on numbers instead of the strings. JavaScript will most likely do it right, but you want to be super clear. 

I'd also recommend putting the alert in the function.